### PR TITLE
BOAC-2132, curated group student-count should not include the group's 'unavailable' SIDs

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/BOAC-2132

Includes more concise wording (eg, toss out "Bulk")

### CODE REVIEW

File: boac/api/util.py [hunk 1]
```diff
@@ -261,14 +261,14 @@ def get_my_curated_groups():
     user_id = current_user.id
     for curated_group in CuratedGroup.get_curated_groups_by_owner_id(user_id):
         _curated_group_api_json(curated_group)
```
Comment (paulkerschen) [inline boac/api/util.py]: @johncrossman Not that this line is part of your PR, but I wonder how it got in here? It doesn't seem to be generating anything that subsequent lines use; looks like it's just building a data structure and throwing it away.

Comment (johncrossman) [reply boac/api/util.py]: You're right and I think it should go. Amending.
